content/en/debugging.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Debugging
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+Debugging is the process or activity of finding and resolving bugs (or errors) from computer programs, software, or systems to get the desired output. A bug is a defect or a problem leading to incorrect or unexpected output.
+
+## Problem it addresses
+Developers write source code and create a binary file with external libraries to run it in production. If a bug is introduced during that process, the code will likely not function as desired (undefined behavior) when executed. To identify the source of the problem, developers have to review the code line-by-line. Catching errors is quite hard without context, i.e., variables, data, location in memory, etc.
```

Comment:
Maybe I am too picky here, but while devs write source code, they don't necessarily create binary files. Sure, apps. run a binary at a very low level, but I think this fact does not add any value to explaining the term. Same goes  for "with external libs". 

Also it is not only about running in production. Debugging is relevant for apps running on any environment, right? 

"To identify the source of the problem, developers have to review the code line-by-line" That is not the whole truth. While it might be true, it reads as if this is the only possibility devs have. We can go through logs as well in order to find what the app is doing, without having to lock at code at all.